tests/ui/target_modifiers/incompatible_min_function_alignment.error_mismatch.stderr

```diff
@@ -0,0 +1,13 @@
+error: mixing `-Zmin-function-alignment` will cause an ABI mismatch in crate `incompatible_min_function_alignment`
```

Comment:
Discussion: when this *isn't* a language guarantee, it's almost like this should be a warning/warn-by-default lint, right?

> the `-Zmin-function-alignment` flag modifies the ABI so Rust crates compiled with different values of this flag cannot be used together safely

When this is a *hint*, then the current target modifier infra/error message might be a bit confusing